src/readii/negative_controls_refactor/__main__.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+
+from rich import print as rprint
+
+from readii.loaders import loadDicomSITK, loadRTSTRUCTSITK
+from readii.negative_controls_refactor.manager import NegativeControlManager  # noqa
+from readii.negative_controls_refactor.negative_controls import (
+	RandomizedControl,
+	SampledControl,  # noqa
+	ShuffledControl,  # noqa
+)  # noqa
+from readii.negative_controls_refactor.regions import FullRegion, NonROIRegion, ROIRegion  # noqa
+
+RANDOM_SEED = 10
+
+
+def main() -> None:
+	"""Test the negative_controls_refactor module."""
+	################################################################################################
+	# Load Data
+	dataset_path = Path.cwd() / "tests" / "4D-Lung"
+
+	ct_dir = (
+		dataset_path
+		/ "113_HM10395/11-26-1999-NA-p4-13296/1.000000-P4P113S303I10349 Gated 40.0B-29543"
+	)
+	rtstruct_dir = (
+		dataset_path
+		/ "113_HM10395/11-26-1999-NA-p4-13296/1.000000-P4P113S303I10349 Gated 40.0B-47.35"
+	)
+	rtstruct_file = next(rtstruct_dir.glob("*.dcm"))
+
+	ct = loadDicomSITK(ct_dir)
```

Comment:
So I know I have this function and I used it in example.ipynb, but the actual pipeline uses the read_dicom_series function from med-imagetools. I feel like we should discuss what to do with the loadDICOMSITK function. I feel like we can either:
1. Get rid of it and just use read_dicom_series everywhere
2. Call read_dicom_series in loadDICOMSITK like how we call the MIT functions in loadRTSTRUCTSITK to load the RTSTRUCT

Thoughts?

Reply:
i never realized that the pipeline uses a different one

I like option #2, gives us opportunity to log in our own functions. 

Lets create a new issue for this

Reply:
Sounds good